ogr/services/github/pull_request.py

```diff
@@ -101,6 +102,17 @@ def labels(self) -> List[GithubLabel]:
     def diff_url(self) -> str:
         return f"{self._raw_pr.html_url}/files"
 
+    @property
+    def patch(self) -> bytes:
+        response = requests.get(self._raw_pr.patch_url)
+
+        if not response.ok:
+            raise GithubAPIException(
+                f"Couldn't get patch from {self._raw_pr.patch_url} because {response.reason}."
+            )
+
+        return response.content
```

Comment:
this is definitely a correct thing to do since we don't know what's inside and might not be able to decode the content